get_started_with_gx_cloud/workshop.md

```diff
@@ -38,12 +38,28 @@ You'll need to setup a data source from GX Cloud. We provide demo data in a Post
 > **Select the `nyc_taxi_data` Data Asset**
 > 1. On the **Select tables to import** page, check the box next to `nyc_taxi_data`.
 > 1. Click **Add 1 Asset**.
+> 1. Click **Start monitoring**.
 
 <img src="img/add_demo_data_asset.png" alt="Add a Demo Data Asset" style="width:600px;"/><br>
+<img src="img/add_demo_data_monitoring.png" alt="Start Monitoring" style="width:600px;"/><br>
 
-Congratulations! You have successfully added the `nyc_taxi_data` demo Data Asset to your GX Cloud organization.
+Congratulations! You have successfully added the `nyc_taxi_data` demo Data Asset to your GX Cloud organization. Click onto the newly-created Data Asset to view it.
 
-## Create Expectations
+<img src="img/view_demo_asset.png" alt="View Demo Data Asset" style="width:600px;"/><br>
+
+## Generate Expectations
```

Comment:
The links in the "Agenda" section above should be updated to reflect this additional task and the changed name of the next task